torch/random.py

```diff
@@ -167,3 +168,7 @@ def fork_rng(devices=None, enabled=True, _caller="fork_rng", _devices_kw="device
         torch.set_rng_state(cpu_rng_state)
         for device, device_rng_state in zip(devices, device_rng_states):
             device_mod.set_rng_state(device_rng_state, device)
+
+# We keep track of all Generator instances (except the default one) via a registry of weak references.
```

Comment:
Ho why not the default one? Because that is the global rng and so it is already handled?